File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/service/model/question/FurtherReadingTest.java
package uk.co.claritysoftware.exam.slammr.webapp.service.model.question;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import uk.co.claritysoftware.exam.slammr.webapp.service.model.question.FurtherReading;

/**
 * Unit test class for {@link FurtherReading}
 */
public class FurtherReadingTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(FurtherReading.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/service/model/question/FurtherReadingTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.service.model.question

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [FurtherReading]
 */
class FurtherReadingTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(FurtherReading::class.java)
				.verify()
	}
}